Modules/Registration/RegistrationMethodsv4/wrapping/itkSyNImageRegistrationMethod.wrap

```diff
@@ -1,12 +1,15 @@
 itk_wrap_include("itkBSplineSmoothingOnUpdateDisplacementFieldTransform.h")
+itk_wrap_include("itkMesh.h")
 
 itk_wrap_class("itk::SyNImageRegistrationMethod" POINTER)
   foreach(d ${ITK_WRAP_DIMS})
     foreach(t ${WRAP_ITK_REAL})
       itk_wrap_template("${ITKM_${t}}${d}${ITKM_${t}}${d}"
                         "itk::Image< ${ITKT_${t}}, ${d} >, itk::Image< ${ITKT_${t}}, ${d} >")
-      itk_wrap_template("${ITKM_${t}}${d}${ITKM_${t}}${d}BSOUDFT${ITKT_D}${d}"
+      itk_wrap_template("${ITKM_${t}}${d}${ITKM_${t}}${d}BSOUDFT${ITKM_D}${d}"
```

Comment:
This change looks like a bug fix. Maybe it warrants a separate commit, with the explanation what was broken?

Reply:
I can create separate commit. Actually nothing is broken. I discovered it while doing the Syn wrapping. It is change in naming only. If there is some code which is directly calling it by that name then it can fail. Instantiating using Class[TypeA, TypeB] will remain the same.